app/models/keyword.rb

```diff
@@ -3,5 +3,7 @@
 class Keyword < ApplicationRecord
   belongs_to :user
 
+  validates :keyword, presence: true, length: { maximum: 255 }
```

Comment:
The column name actually should have been as `name` instead of `keyword` as the model/table name is `Keyword` 😄 . I am fine with the current changes, if you want you can add a migration to change the column name

Reply:
Good point! Noted! I will update this in another PR to no increase the size of this PR